wishlistApp/templates/wishlistApp/userList.html

```diff
@@ -16,11 +16,15 @@
           <th>{{items.image}} </th>
           <th>{{items.description}} </th>
           <th>{{items.priority}} </th>
-          <th> <button name="Edit" style="background:green">Edit</button>
+          <th> <button name="Edit" style="background:green">
+                <a href="% url 'add-item' %">Edit</a>
+                </button>
                <button name="Remove" style="background:red">Remove</button></th>
         </tr>
       {% endfor %}
-      <button name="Add"> Add Item </button>
+      <button name="Add">
+        <a href="% url 'add-item' %"> Add Item </a>
```

Comment:
this url doesn't work. there's no `add-item` url in our urls.py. I think what you meant was `newItem`. The name in the `{% url '' $}` should be the name you set the url to be in the urls.py file, not the actual link endpoint itself.